jupiterone/resource_rule.go

```diff
@@ -276,6 +346,13 @@ func (*QuestionRuleResource) Schema(ctx context.Context, req resource.SchemaRequ
 			"labels": schema.ListNestedAttribute{
 				Description: "Comma separated list of labelName/labelValue pairs to apply to the rule.",
 				Optional:    true,
+				Computed:    true,
+				PlanModifiers: []planmodifier.List{
```

Comment:
This allows for minimize terraform plan diffs if the data comes back as an empty list from the api.

This allows the nil type to be converted to: [] so we can minimize diffs unless absolutely necessary.